src/Component.js

```diff
@@ -41,11 +39,11 @@ export class CopyToClipboard extends React.PureComponent {
       text: _text,
       onCopy: _onCopy,
       options: _options,
+      onClickCapture,  // eslint-disable-line
       children,
-      ...props
+      ...restProps
     } = this.props;
-    const elem = React.Children.only(children);
 
-    return React.cloneElement(elem, {...props, onClick: this.onClick});
+    return React.cloneElement(children, {...restProps, onClick: this.onClick});
```

Comment:
Well that's the point. We explicitly work with the only child, not an array. This is why I am checking what is going to happen when array is passed (don't really remember from the top of my head)